service/src/test/java/uk/gov/hmcts/reform/fpl/model/HearingCourtBundleTest.java

```diff
@@ -0,0 +1,113 @@
+package uk.gov.hmcts.reform.fpl.model;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.unwrapElements;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {JacksonAutoConfiguration.class})
+public class HearingCourtBundleTest {
+    private static final UUID TEST_ID = UUID.randomUUID();
+    private static final String TEST_HEARING = "Test hearing";
+    private static final String CONFIDENTIAL = "CONFIDENTIAL";
+
+    private static final CourtBundle CONFIDENTIAL_COURT_BUNDLE = CourtBundle.builder()
+        .document(DocumentReference.builder()
+            .binaryUrl("binaryUrl")
+            .filename("filename")
+            .url("url")
+            .build())
+        .confidential(List.of(CONFIDENTIAL))
+        .build();
+
+    private static final CourtBundle NON_CONFIDENTIAL_COURT_BUNDLE = CourtBundle.builder()
+        .document(DocumentReference.builder()
```

Comment:
I think there's a testDocumentReference() function which can skip all this stuff